File: 0242-valid-anagram/0242-valid-anagram.java
class Solution {
    public boolean isAnagram(String s, String t) {
        if(s.length() != t.length()){
		    return false;
		} else {
		    int[] array = new int[26];
		    for(char ch : s.toCharArray())
		        array[ch-'a']++;
		    for(char ch : t.toCharArray())
		        array[ch-'a']--;
		    for(int ele : array)
		        if(ele != 0)
		            return false;
		    return true;
		}
    }
}

File: 0242-valid-anagram/0242-valid-anagram.py
class Solution:
    def isAnagram(self, s: str, t: str) -> bool:        
        # Using Arrays
        if(len(s) != len(t)):
            return False
        else:
            array = [0 for x in range(26)]
            for char in s:
                array[ord(char)-97]+=1
            for char in t:
                array[ord(char)-97]-=1
            for ele in array:
                if ele != 0:
                    return False
            return True